steps/property-inspection/garden.tsx

```diff
@@ -105,6 +105,43 @@ const step = {
           })
         })
       ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "is-maintained",
+          Component: RadioButtons,
+          props: {
+            name: "is-maintained",
+            legend: (
+              <FieldsetLegend>Is the garden being maintained?</FieldsetLegend>
+            ) as React.ReactNode,
+            radios: [
+              {
+                label: "Yes",
+                value: "yes"
+              },
+              {
+                label: "No",
+                value: "no"
+              }
+            ]
+          },
+          renderWhen(stepValues: {
+            "garden-type"?: ComponentValue<ProcessDatabaseSchema, "property">;
+          }): boolean {
+            return stepValues["garden-type"] === "not sure";
+          },
+          defaultValue: "",
+          emptyValue: "",
+          databaseMap: new ComponentDatabaseMap<
+            ProcessDatabaseSchema,
+            "property"
+          >({
+            storeName: "property",
+            key: processRef,
+            property: ["garden", "isMaintained"]
+          })
+        })
+      ),
```

Comment:
Looks like you duplicated this component. I would have expected this to be an edit of the existing component?

Reply:
Yes. Wasn't sure how to edit a component with renderWhen both "not sure" and "no" as using && and || was throwing errors.